### ISSUE TITLE
PHK need D-bus to communicate with avahi-daemon ?

### ISSUE THREAD

**Issue by GharsallaouiNader** [etwmc/Personal-HomeKit-HAP-#41]
if i change in avahi-daemon.conf "enable-dbus=yes" to "enable-dbus=no" the IPhone don't find the PHK
and there are no error message 
i need to now if the PHK use d-bus to communicate with avahi-daemon

**Comment by etwmc** [comment]
For the Bonjour broadcast, I access you will need dbus. Because judging from what I can find from Google, it's used for the software to the daemon, which actually broadcast the service.

**Comment by GharsallaouiNader** [comment]
the broadcast MDNS of bonjour work fine the problem is that the service added by the PHK DNSServiceRegister() don't work. i use wireshark to verify

**Comment by etwmc** [comment]
Yeah. You need service register to attach service to the broadcast.
